src/main/java/com/example/repository/AdministratorRepository.java

```diff
@@ -75,11 +75,9 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 	 * @param administrator 管理者情報
 	 */
 	public void insert(Administrator administrator) 
-	{
-	SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
+	{SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
 		String sql = "insert into administrators(name,mail_address,password)values(:name,:mailAddress,:password);";
-		template.update(sql, param);
-	}
+		template.update(sql, param);}
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>';' is not followed by whitespace.